index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Please, it will be nice if you include the lang attribute because the lang attribute specifies which language is used to write the content of a web page. It is used to set the language for the whole text of the web page.

Reply:
Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions

Reply:
Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions.